usecases/events/retrieve_aggregate_event_impact.py

```diff
@@ -8,15 +8,17 @@
 phq = Client(access_token=ACCESS_TOKEN)
 
 
-# The PredictHQ API offers two types of aggregation on events.
+# The PredictHQ Aggegate Event Impact API can be used to find the correlation between your demand and the events data.
+# The API offers two types of aggregation on events which you can specify in impact_rank param.
+# You will need Data License to accesss this endpoint and Aviation Rank subscription for Aviation Rank impact.
 
 # You can either aggregate on PHQ Rank
 # https://developer.predicthq.com/resources/events/#param-impact-rank
-for event in phq.events.impact(active__gte="2015-12-24", active__lte="2015-12-26", impact_rank="rank"):
-    print(event.rank_levels, event.rank_levels_impact, event.categories, event.categories_impact)
+for impact_day in phq.events.impact(active__gte='2015-12-24', active__lte='2015-12-26', impact_rank='rank'):
+    print(impact_day.date, impact_day.rank_levels, impact_day.rank_levels_impact, impact_day.categories, impact_day.categories_impact)
```

Comment:
I think it would be good to add the `impact_day.impact` field in the example here and below.